GsaGHTests/Parameters/VectorResultGooTest.cs

```diff
@@ -122,9 +125,9 @@ public void WhenGetTypeDescription_ThenShouldReturnValidString()
     [Fact]
     public void WhenDuplicateGeometry_ThenMethod_ShouldReturnDuplicatedObject()
     {
-      var obj = new VectorResultGoo(Point3d.Origin, Vector3d.Zero, new Force(4, ForceUnit.Kilonewton));
+      var obj = new VectorResultGoo(Point3d.Origin, Vector3d.Zero, new Force(4, ForceUnit.Kilonewton), 0);
 
-      var expectedObj = new VectorResultGoo(Point3d.Origin, Vector3d.Zero, new Force(4, ForceUnit.Kilonewton));
+      var expectedObj = new VectorResultGoo(Point3d.Origin, Vector3d.Zero, new Force(4, ForceUnit.Kilonewton), 0);
```

Comment:
when duplicating, also id will be the same - we need to check it

Reply:
yes, covered by other tests though?